packages/cli/src/commands/base-command.ts

```diff
@@ -97,6 +98,7 @@ export abstract class BaseCommand extends Command {
 		}
 
 		Container.get(ModuleRegistry).initializeModules();
+		Container.get(ModuleRegistry).registerMultiMainListeners(Container.get(MultiMainSetup));
```

Comment:
Should we extract `Container.get(ModuleRegistry)` into a private field or a variable ?